docs/platform-setup.md

```diff
@@ -18,3 +18,126 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Configure Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Configuring docker registry with nexus should follow this steps:
+
+### Create New Docker Repository
+
+Open the platform team's [Nexus](https://nexus.platform.dev.nuxeo.com/). Start new session using admin user
+
+#### Blob Storage And Repository Configuration
+
+- Create a blob storage : `docker-registry`
+
+  - In Server Administration and Configuration > Blob Stores,  click on the "Create blob store" button
```

Comment:
```suggestion
  - In Server Administration and Configuration > Repository > Blob Stores,  click on the "Create blob store" button
```

Reply:
Hi,

Blob storage creation button is not under Repository ???

Reply:
changes applied